example/src/main/java/com/dynatrace/opentelemetry/metric/example/DynatraceExporterExample.java

```diff
@@ -55,20 +59,22 @@ public static void main(String[] args) throws Exception {
     // Create a counter
     LongCounter counter =
         meter
-            .longCounterBuilder("example_counter")
+            .longCounterBuilder("otel.dynatrace.java.example_counter")
```

Comment:
It's `otel.python.*` in our Python exporter

```suggestion
            .longCounterBuilder("otel.java.example_counter")
```